### ISSUE TITLE
Colon and Lung Cancer Type Prediction using Deep Learning

### ISSUE THREAD

**Issue by adi271001** [abhisheks008/DL-Simplified#375]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Colon and Lung Cancer Prediction Type using Deep Learning </br>
:red_circle: **Aim** : To create a Deep Learning model which predicts the class  of lung and colon cancer</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/andrewmvd/lung-and-colon-cancer-histopathological-images/</br>
:red_circle: **Approach** : Modeil is built using Deep CNN algorithm it has 13 CNN Layers 5 Max pooling Layers and 3 Dense Layers ReLU and softmax have been used as activators and adamax has been used as optimizer.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name :  Aditya D
- GitHub Profile Link : https://www.github.com/adi271001
- Email ID : ibetanibex@gmail.com
- Participant ID (if applicable):
- Approach for this Project :
1. Import Libraries 
2. Perform Preprocessing of Data
3. Creating all the required functions 
4. Split Data into test and train
5. Build the DeepCNN model 
6. Train the model
7. Predict on test data
8. Recording the accuracy via plots like confusion matrix
9. Saving the Model and predicition results in csv 
- What is your participant role? (Mention the Open Source program) OSEN Participant

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by abhisheks008** [comment]
Issue assigned to you @adi271001. All the best and go ahead!

**Comment by adi271001** [comment]
@abhisheks008 i'm not able to push my changes it's giving permission denied

**Comment by adi271001** [comment]
I have made a pull request
